spec/unit/hanami/events/adapter/memory_spec.rb

```diff
@@ -1,15 +1,15 @@
 RSpec.describe Hanami::Events::Adapter::Memory do
   let(:adapter) { described_class.new }
 
-  describe '#subscribe' do
-    it 'pushes listener to listener list' do
+  context '#subscribe' do
```

Comment:
Should I follow betterspecs and use shared_examples when needed? :trollface: 
Just kidding, ok, changing context to describe right now.